src/providers/bedrock/complete.ts

```diff
@@ -754,3 +789,124 @@ export const BedrockCohereCompleteStreamChunkTransform: (
     ],
   })}\n\n`;
 };
+
+interface BedrocMistralStreamChunkOutput {
+  text: string;
+  stop_reason: string | null;
+}
+
+export interface BedrocMistralStreamChunk {
+  outputs: BedrocMistralStreamChunkOutput[];
+}
+
+export const BedrockMistralCompleteStreamChunkTransform: (
+  response: string,
+  fallbackId: string
+) => string | string[] = (responseChunk, fallbackId) => {
+  let chunk = responseChunk.trim();
+  chunk = chunk.trim();
+  const parsedChunk: BedrocMistralStreamChunk = JSON.parse(chunk);
+
+  if (parsedChunk.outputs[0].stop_reason) {
+    return [
+      `data: ${JSON.stringify({
+        id: fallbackId,
+        object: 'text_completion',
+        created: Math.floor(Date.now() / 1000),
+        model: '',
+        provider: BEDROCK,
+        choices: [
+          {
+            text: parsedChunk.outputs[0].text,
+            index: 0,
+            logprobs: null,
+            finish_reason: parsedChunk.outputs[0].stop_reason,
+          },
+        ],
+        usage: {
```

Comment:
Bedrock sends token usage in the last chunk of the stream. I am not sure if that is the case for Mistral as well. Can you please check and  make the change accordingly. Here is an example: https://github.com/Portkey-AI/gateway/blob/48785807bcc19dc13669354ac3bc895b9af58185/src/providers/bedrock/complete.ts#L604-L612

Reply:
yup you are riight, they do send the response as well. will update